Sources/SmileID/Classes/Helpers/LocalStorage.swift

```diff
@@ -2,7 +2,9 @@ import Foundation
 import Zip
 
 public class LocalStorage {
-    private static let defaultFolderName = "sid_jobs"
+    private static let defaultFolderName = "SmileID"
+    private static let pendingFolderName = "pending"
+    private static let completedFolderName = "completed"
```

Comment:
As we discussed today, we should start calling everything as submitted/unsubmitted rather than pending/completed